### DISCUSSION TITLE
Markwhen format?

### DISCUSSION THREAD

**Discussion by jzb** [juliaferraioli/opensource-timeline#20] [Ideas]
Not long ago I stumbled on "Markwhen," a markdown-like format for creating timelines. (See: [Markwhen website](https://markwhen.com/). I've been playing with that for creating some timelines for a series I've been writing about Red Hat's "clone wars." I wonder if it'd be possible to consider that format before there's too many entries in this repo to consider a switch? It seems more friendly / easier to contribute.

**Comment by juliaferraioli** [comment]
Ha! That's a great suggestion, and in fact one I'm working on at the moment.

I'm a bit delayed with other work right now, but the thing I'm trying to get right is the linting so that a schema is still enforced.

I couldn't find an existing way to do that so I'm implementing custom rules in `mdl` and my ruby is rusty.

But yes, this is definitely the right direction and much more user-friendly.

**Reply by juliaferraioli** [reply]
Argh, that's what I get for replying when I just wake up.

I haven't seen `markwhen` before, but I'll definitely check it out.

I had been leaning towards an approach of a static site generator with one file per event,  but let me see if `markwhen` is a better way to go!

**Reply by jzb** [reply]
Thanks! Happy to help with porting things over if you decide to go in that direction. My programming skills are minimal, but I can edit text with the best of 'em...
